README.md

```diff
@@ -24,14 +24,11 @@
 - [🙏 Acknowledgements](#acknowledgements)
 - [📝 License](#license)
 
-# 📖Bookstore: add reducers and actions<a name="about-project"></a>
+# 📖Bookstore: Use Redux in React components<a name="about-project"></a>
 
-**Bookstore: add reducers and actions**
+**Bookstore: Use Redux in React components**
 
-In this step, you will implement Redux Toolkit inside of a React application. You will:
-
-- Setup a Redux store
-- Create a slice to save your state and reducers
+In this step you will use a Redux store to display books. You'll implement react-redux in order to connect your store and then access your state variables (accessed through useSelector) and state modifiers (which will be a combination of a reducer and useDispatch).
 
```

Comment:
- [ ] Live demo link on _line 52_ of your ReadMe is broken, kindly endeavor to fix or remove it if this app is not yet deployed. Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Reply:
Thank you for helping me make the project more professional. I'm confident that everything is now in line with your suggestions